### DISCUSSION TITLE
How can I add data in the `taggables` table

### DISCUSSION THREAD

**Discussion by Fredericld** [spatie/laravel-tags#292] [Q&A]
Hi,

I need to add a score against the tags allocated to my users.
I have added a `score` column in the `taggables` table but I am really not sure how I feed it or read it.

I can not see anything in the doc to do this kind of action. is it even possible?

**Comment by neverything** [comment]
Very late to the party, but according to the test in https://github.com/spatie/laravel-tags/blob/main/tests/TestClasses/TestCustomTagModel.php you can create a custom tag model and overwrite which table is used
